### PR TITLE
Add integration tests for null CORS parameters 

### PR DESCRIPTION
**Pull request by imsuneth**
Integration tests for the fix: https://github.com/wso2-support/carbon-apimgt/pull/5407

### CODE REVIEW

**Review by msm1992 — APPROVED** [review]

File: modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/publisher/CheckEmptyCORSConfigurationsTestCase.java [hunk 1]
```diff
@@ -0,0 +1,136 @@
+/*
+ *
+ *   Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (tharikaGitHub) [inline modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/publisher/CheckEmptyCORSConfigurationsTestCase.java:3]: Shall we change the license header to WSO2 LLC.

Comment (imsuneth) [reply modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/publisher/CheckEmptyCORSConfigurationsTestCase.java]: Changed https://github.com/wso2/product-apim/pull/12982/commits/89c0c1cb6ac7821d68a10e25d866de5b63fa815c

**Review by tharikaGitHub — APPROVED** [review]
